synapsis/src/main/resources/application.properties

```diff
@@ -8,4 +8,7 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.properties.hibernate.format_sql=true
 spring.sql.init.mode=always
-spring.jpa.defer-datasource-initialization=true
\ No newline at end of file
+spring.jpa.defer-datasource-initialization=true
+spring.security.oauth2.client.registration.google.client-id=337739041995-a0nn8plaogqpfp9s3d3ebevur0fksfak.apps.googleusercontent.com
```

Comment:
just double-checking to avoid security risk, client-id can be public and the secret is handled through secrets.properties?

Reply:
Yes, exactly.